### PR TITLE
Delete changelog transient when saving the RCP download. #49

### PR DESCRIPTION
**Pull request by mindctrl**
#49

### CODE REVIEW

File: includes/theme-tweaks.php [hunk 1]
```diff
@@ -320,6 +320,16 @@ function rcp_get_changelog() {
 }
 
 /**
+ * Clears the changelog transient when the RCP download is saved.
+ */
+function rcp_delete_changelog_transient( $post_id ) {
+	if ( 479 == $post_id ) {
```
Comment (sumobi) [inline includes/theme-tweaks.php]: Let's use `rcp_get_download_id()` instead of the post ID (handy for local development)

**Comment by sumobi** [comment]
@mindctrl Good deal, need to do this for affwp site also. Left a minor comment.

**Comment by mindctrl** [comment]
@sumobi didn't know we had that function. That's slick. Updated the PR.

**Comment by sumobi** [comment]
deployed

**Pull request merged**